common/religion/doctrines/00_core_tenets.txt

```diff
@@ -1824,7 +1825,7 @@
 		
 		# Warcraft
 		is_shown = {
-			tenet_moon_worship_allowed_trigger = no
+			always = no
```

Comment:
I don't think astrology should be removed completely.
It could be kept as a cheaper alternative or for faiths where arcane scrying doesn't fit(like those who shun the arcane).

Reply:
would you be OK with renaming it something like "augury" so that it kind of denotes superstitious fortune telling, because astrology itself doesnt really work in the warcraft setting